Lesson2/src/main/java/fc/introduction/calculator/Calculator.java

```diff
@@ -0,0 +1,20 @@
+package fc.introduction.calculator;
+
+public class Calculator {
+    
+    public double sum(double firstNumber, double secondNumber) {
+        return firstNumber + secondNumber;
+    }
+
+    public double subtraction(double firstNumber, double secondNumber) {
```

Comment:
Nazewnictwo jest niespójne. Metody powinniśmy nazywać czasownikami (tak jak w przypadku `sum` i `divide` - zsumuj, podziel). `Substraction` oznacza odejmowanie - zmieńmy więc to na `subtract` - odejmij. Wtedy będziemy mieli spójność z nazewnictwem innych metod. 

W programowaniu trzeba bardzo uważać na spójność w nazewnictwie. Jeśli mamy 4 metody i 2 z nich są nazwane czasownikiem, a 2 rzeczownikiem, to jest to dosyć niekonsekwentne. Powinniśmy wszystkie nazwać czasownikami